tests/phpunit/tests/option/determineOptionAutoloadValue.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @group option
```

Comment:
See my other comment below:
```suggestion
 * @group option
 *
 * @covers ::determine_option_autoload_value
```